handsontable/src/plugins/copyPaste/copyPaste.js

```diff
@@ -566,24 +554,22 @@ export class CopyPaste extends BasePlugin {
     this.setCopyableText();
     this.#isTriggeredByCopy = false;
 
-    const data = this.getRangedData(this.copyableRanges);
-    const copiedHeadersCount = this.#countCopiedHeaders(this.copyableRanges);
```

Comment:
The `#countCopiedHeaders` method is no longer used within the class, so it can be removed (line 458).